.github/workflows/run-archiver.yml

```diff
@@ -15,12 +15,22 @@ jobs:
       matrix:
         dataset:
           - eia176
+          - eia181
           - eia191
           - eia757a
           - eia860
           - eia860m
           - eia923
           - eia_bulk_elec
+          - eiawater
+          - ferc1
+          - ferc2
+          - ferc6
+          - ferc60
+          - ferc714
```

Comment:
Do we still expect the FERC XBRL archives will always appear completely new, even if nothing has changed in their contents?

IIRC they were doing something like autogenerating new IDs for every post in the RSS feed every time the feed was read, rather than using persistent unique IDs for each post. Did we find some way around that?

Reply:
FERC archivers are certainly not working at present, but I'm tracking this in #285.

Reply:
Hmm, #285 looks like real brokenness in the FERC archivers.

If we haven't addressed the unique ID thing, we'd just see that all the FERC XBRL archives get updated every time the archiver is run, but I think we'd still be able to get an idea of how much new data there is from the change in the size of the archives, and saving interim data doesn't seem like a bad idea given how flaky FERC's data curation is!